.github/workflows/test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored
```

Comment:
Your `test.md` file needs to be moved out of this folder, that is the reason why it is not being ignored by `.gitignore`. I recommend you make some research about the following Git command: `git rm -r --cached`, you might need it 😉

Reply:
Done